playbook/app/pb_kits/playbook/pb_form_pill/_form_pill.scss

```diff
@@ -16,18 +17,24 @@ $form_pill_colors: (
   display: inline-flex;
   justify-content: center;
   align-items: center;
-  padding: 0 calc($space-sm/3);
+  padding: 0 calc($space-md/2);
```

Comment:
I know we should use our tokens as much as possible, but this is still a bit strange to me. I'd prefer 12px.

Reply:
That is a good point for sure. I was keeping with the pattern found throughout [this scss file](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_form_pill/_form_pill.scss) and a few other scss files - I will make a note to double check with the team on the current feeling about this token-calc pattern and from there consider revamping the file in the follow up form pill story (which I should be picking up in a sprint or two). Thanks for the note!